.gitignore

```diff
@@ -13,3 +16,108 @@ __pycache__
 .classpath
 .settings
 target
+
+### Intellij ###
```

Comment:
I guess so. I only left it in because it was generated by gitignore.io and so I thought it might be helpful for excluding other Intellij files in future. However, I'm happy to remove it as we don't need it currently.

Reply:
done